api/src/routes/settings.ts

```diff
@@ -163,6 +163,55 @@ export const settingRoutes: FastifyPluginCallbackTypebox = (
     }
   );
 
+  fastify.put(
+    '/update-my-about',
+    {
+      schema: {
+        body: Type.Object({
+          about: Type.Optional(Type.String()),
+          name: Type.Optional(Type.String()),
+          picture: Type.Optional(Type.String()),
```

Comment:
Can't we use `{ format: 'uri' }` here?

Reply:
It won't be optional, even when validation saying it optional, if we add format, it has to follow it.

If we add format url, then we need to remember to stop sending empty strings if user doesn't update the picture in the client.

Reply:
Ah, good point, it changes the spec.

Sorry, @Sboonny, you were right - `Type.Optional(Type.String())` is the best we can do.